ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -14,12 +14,8 @@ parameters:
   required: true
 - name: LOCAL_IMAGE_LIBVIRT_INSTALLER
   required: true
-- name: IMAGE_TESTS
-  required: true
 - name: CLUSTER_TYPE
   value: "libvirt"
-- name: TEST_COMMAND
-  required: true
```

Comment:
Does @smarterclayton need this for some test-consistency reason?  We could keep this and set it `true` (the POSIX command) until we're ready to return to e2e on libvirt.

Reply:
Not sure, locally this runs fine with it omitted.  When we're ready to enable openshift/smoke-4 suite here, we'll return to this commit, to put back what this PR removes (or, I could leave it in commented out with a note?) Either way, this job is useful, since this past week alone, the libvirt setup was broken(or docs not updated) several times (missing TAG, terraform plugin issues, etc) and those would've been easier to diagnose/clear up with this job.  Even w/out the e2e tests, this ensures the libvirt setup produces a cluster functioning to the point of worker/routes/registry/router up and running.

Reply:
I agree this job is useful.  I just have fuzzy memories of @smarterclayton os @stevekuznetsov telling me to keep a variable here for consistency with other templates.  I dunno if that applies to `TEST_COMMAND`.